groceries_app/static/css/recipes/recipes-detail.css

```diff
@@ -0,0 +1,72 @@
+pre {
+    padding: 20px;
+    border: 0;
+    font-family: "Neucha", -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+    font-size: 18px;
+    letter-spacing: 1px;
+}
+
+.sketchy-font {
+    font-family: "Neucha", -apple-system, system-ui, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
+    font-size: 18px;
+    letter-spacing: 1px;
```

Comment:
Casual inquiry: Is this style unique to this page? If not, can css styles be inherited from a base class?

Reply:
So far, this style including the font size and letter spacing is unique to this page, however the font family is not.